tests/browser-tests.js

```diff
@@ -0,0 +1,57 @@
+/* globals describe, before */
+
+// const expect = require('chai').expect
+const async = require('async')
+const store = require('local-storage-blob-store')
+const tests = require('./repo-test')
+const _ = require('lodash')
+const IPFSRepo = require('../src')
+
+global.Buffer = require('buffer').Buffer
```

Comment:
Take a look [here](https://webpack.github.io/docs/configuration.html#node)

Adding `node.Buffer = true` should do the trick